<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesh visualization</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>261</width>
         <height>492</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1,1,1,0,0,0,0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>About Crust algorithm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckTriangles">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Show triangles</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckCells">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Show Voronoi cells</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckPoints">
          <property name="text">
           <string>Show data points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckCenters">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Show Voronoi centers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckCurve">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Show reconstructed curve</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckCircles">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Show circles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Save features</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveOff">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Save to OFF format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Information :</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To move the view, use the mouse :&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; - Keep pressing left click to move the figure.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; - Keep pressing right click to rotate the figure&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; - Scroll up or down the wheel button to zoom.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ReloadButton">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Reload</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="GLDisplay" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Load</string>
     </property>
     <addaction name="actionOFF_mesh"/>
     <addaction name="actionPTS_triangulation"/>
     <addaction name="actionPTS_curve"/>
     <addaction name="action2D_Constraint_Triangulation"/>
    </widget>
    <addaction name="menuLoad"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSubmenu">
   <property name="text">
    <string>submenu</string>
   </property>
  </action>
  <action name="actionOFF_mesh">
   <property name="text">
    <string>3D Mesh</string>
   </property>
  </action>
  <action name="actionPTS_triangulation">
   <property name="text">
    <string>2D Triangulation</string>
   </property>
  </action>
  <action name="actionPTS_curve">
   <property name="text">
    <string>2D Crust triangulation</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action2D_Constraint_Triangulation">
   <property name="text">
    <string>2D Constraint Triangulation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLDisplay</class>
   <extends>QWidget</extends>
   <header location="global">gldisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
